compiler/qsc/src/interpret/debug/tests.rs

```diff
@@ -87,7 +87,7 @@ fn stack_traces_can_cross_eval_session_and_file_boundaries() {
                              at Adjoint Test.C in 1.qs
                              at Adjoint Test.B in 1.qs
                              at Adjoint Test2.A in 2.qs
-                             at Z in <expression>
+                             at Z in line_0
```

Comment:
This behavior change results from the fact that we're now keeping the HIR updated in the package store, allowing [`get_item_file_name`](https://github.com/microsoft/qsharp/blob/30f11ca118ba39a59f76be2adecf57f08d80ff90/compiler/qsc/src/interpret/debug.rs#L74) to work for the current package.